app/src/renderer/components/network/PageBlock.vue

```diff
@@ -30,9 +32,12 @@
         <div class="row">
```

Comment:
should we `v-if="!block.block.evidence"`?

Reply:
what is the evidence used for in the first place ?

Reply:
deleted evidence